### PR TITLE
log training loss in pytorch trainer

### PR DESCRIPTION
**Pull request by bw4sz**
With the updates to pytorch lightning 2.0, we appear to no longer to auto log the training loss! This is kinda a big deal, and took me by surprise. 

We previously got the training loss every epoch automatically.

Simple fix is to add self.log within training_step.

### CODE REVIEW

**Review by ethanwhite — APPROVED** [review]

**Comment by ethanwhite** [comment]
Looks like the style tests are flagging an issue. Once that's fixed this is ready to go.

**Review by ethanwhite — APPROVED** [review]
